privacyidea/lib/smsprovider/HttpSMSProvider.py

```diff
@@ -145,10 +147,11 @@ def submit_message(self, phone, message):
             params = {}
             data = parameter
 
-        log.debug("issuing request with parameters %s and method %s and "
-                  "authentication %s to url %s." % (parameter, method,
+        log.debug("issuing request with parameters %s, headers %s and method %s and "
+                  "authentication %s to url %s." % (params, headers, method,
```

Comment:
Please use the {0!s} notation.

   u"issuing request with parameters {0!s}, headers {1!s}and method {2!s} and "
                  "authentication {3!s} to url {4!s}." % (params, headers, method,